docs/documentation/pages/DECKHOUSE-FAQ.md

```diff
@@ -156,6 +156,46 @@ You need to use the Proxy Cache feature of a [Harbor](https://github.com/goharbo
 
 Thus, Deckhouse images will be available at `https://your-harbor.com/d8s/deckhouse/{d8s-edition}:{d8s-version}`.
 
+### Manually upload images to an air-gapped registry
+
+Download script on a host that have access to `registry.deckhouse.io`. `Docker`, `crane` and `jq` must be installed on this host.
+
+```shell
+curl -fsSL -o d8-pull.sh https://raw.githubusercontent.com/deckhouse/deckhouse/main/tools/release/d8-pull.sh
+chmod 700 d8-pull.sh
+./d8-pull.sh --license YOUR_DECKHOUSE_LICENSE_KEY --output-dir /your/output-dir/
+```
+
+Example of pulling images
+
+```shell
+./d8-push.sh --source-dir /your/source-dir/ --path your.private.registry.com/deckhouse --username YOUR_USERNAME --password YOUR_PASSWORD
```

Comment:
```suggestion
./d8-pull.sh --source-dir /your/source-dir/ --path your.private.registry.com/deckhouse --username YOUR_USERNAME --password YOUR_PASSWORD
```

I'm not sure, but the d8-pull.sh script was downloaded.

Reply:
You're right, but not only command is wrong. Will fix it in separate commit.